### PR TITLE
changes 1 reagent color

### PR DESCRIPTION
**Pull request by robintime**
why why why why
why should we have to live with this? why should we endure something so disrespectful?
look guys, i know this PR is uncharted territory. i understand if i don't have your full support on this change.
i just hope that the world is right enough for my vision to finally be realized.

:cl:
 * tweak: egg yolks are now yellow
color

### CODE REVIEW

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -4263,7 +4263,7 @@
 	description = "A chicken before it could become a chicken."
 	nutriment_factor = 15 * REAGENTS_METABOLISM
 	reagent_state = REAGENT_STATE_LIQUID
-	color = "#FFFACD" //LEMONCHIFFON
+	color = "#ffcd42"
```
Comment (TGFH) [inline code/modules/reagents/Chemistry-Reagents.dm:4266]: This should be #03fc30

Comment (robintime) [reply code/modules/reagents/Chemistry-Reagents.dm:4266]: remove your green eggs and ham propaganda from the site of my holy mission

**Comment by Kurfursten** [comment]
🆑 
* rscdel: yolk reagent color 2D green
* rscdel: yolk reagent color 8B blue

**Comment by Eneocho** [comment]
Remember to uncomment the egg yolk from the wet/dry vacuum code.

**Pull request merged**